### DISCUSSION TITLE
France maps missing.

### DISCUSSION THREAD

**Discussion by bigfoot38** [osmandapp/OsmAnd#14073] [Q&A]
Hello,

I recently re-installed Osmand but I could not download all the maps from France.
In the download menu, I navigate to Europe->France and only found 4 maps. When I click on a region of France on the map, the region is highlighted but Osmand does not propose to download a map, while it does if I click on another country.

I think that it is a bug, but do you have another explanation?

Regards

**Comment by vshcherb** [comment]
No explanations so far, probably you can attach screenshots

**Comment by bigfoot38** [comment]
As you can see on the picture below, only 4 maps are available (out of 13). Notice that I moved Osmand from the internal phone memory to the SD memory before noticing the issue. Could it be related to this move operation?

After your answer, I completely reinstalled Osmand, and all the maps are listed now. So the issue just… vanished.

![Screenshot_2](https://user-images.githubusercontent.com/51486078/167253196-b6e8ebeb-0c34-43be-9a01-f948b850a484.png)

_Edit: put a smaller screenshot_

**Reply by vshcherb** [reply]
That's indeed very strange, probably these maps went to Other Maps which yo ucan find on the main screen of Downloads
